src/components/CaptureMatching/CaptureMatchingView.js

```diff
@@ -165,21 +202,33 @@ function CaptureMatchingView() {
             imgCount={imgCount}
             handleSkip={handleSkip}
           />
-
-          <Box style={{ width: '50%' }}>
-            <Box className={classes.candidateIconBox}>
-              <CurrentCaptureNumber
-                text={`Candidate Match${treesCount !== 1 && 'es' || ''}`}
-                treeIcon={treeIcon}
-                treesCount={treesCount}
-              />
-            </Box>
-            <CandidateImages
-              candidateImgData={candidateImgData}
-              sameTreeHandler={sameTreeHandler}
+        </Paper>
+        <Box className={classes.box2} ref={refBox}>
+          {floatTreeIcon && (
+            <Fab
```

Comment:
Is this just meant to be a scroll-to-top button? For me, it's visible all the time and it's meaning isn't clear from the tree count.

Reply:
I've just spotted your comment in the description.
I don't think we need to keep the number displayed – it's just a cue for the user of how many candidates there are for the capture. It can safely disappear when we scroll.

The scroll-to-top button should disappear when you reach the top of the page.